paymentsheet-example/src/main/java/com/stripe/android/paymentsheet/example/playground/settings/PlaygroundSettingDefinition.kt

```diff
@@ -52,6 +59,15 @@ internal interface PlaygroundSettingDefinition<T> {
         }
     }
 
+    interface PlaygroundConfigurable<T> {
+        fun convertToValue(configurationData: PlaygroundConfigurationData): T
+
+        fun updateConfigurationData(
+            value: T,
+            configurationData: PlaygroundConfigurationData
+        ): PlaygroundConfigurationData
```

Comment:
Can we make this return nothing, and pass in a builder instead? That's more similar to how we do others.